### PR TITLE
Readme info for maximum fragment length

### PR DESCRIPTION
**Pull request by muneebahmed10**
*Description of changes:*
Adds some information in mbedtls README about the max fragment length extension

By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.

### CODE REVIEW

**Review by yourslab — APPROVED** [review]

File: external_libs/mbedTLS/README.txt [hunk 1]
```diff
@@ -3,3 +3,7 @@
 # You'll need to download mbedTLS from the official ARMmbed repository and
 # place the files here. We recommend that you pick the latest version of 2.16
 # LTS release in order to have up-to-date security fixes.
+
+# Note that this SDK will not negotiate the TLS Maximum Fragment Length even if
+# it is enabled in mbedTLS' configuration. Therefore, you should ensure content
+# buffers used by mbedTLS are at least 16384 bytes in length.
```
Comment (aggarw13) [inline external_libs/mbedTLS/README.txt:9]: May be, providing more context about the reason of suggesting the size will be helpful. (Feel free to re-word my suggestion for something more appropriate)
```suggestion
# buffers used by mbedTLS are at least 16384 bytes in length to use the largest maximum fragment length supported by TLS.
```

Comment (muneebahmed10) [inline external_libs/mbedTLS/README.txt:9]:
```suggestion
# buffers used by mbedTLS are at least 16384 bytes in length to use the largest
# maximum record length supported by TLS.
```

Comment (muneebahmed10) [reply external_libs/mbedTLS/README.txt]: I like "record length" slightly more so I'll go with that

**Comment by yourslab** [comment]
Hey reviews not dismissed. Good times.

**Review by n9wxu — APPROVED** [review]

**Pull request merged**